code/modules/projectiles/ammo_datums.dm

```diff
@@ -1249,19 +1249,19 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 	accuracy_var_low = 3
 	accuracy_var_high = 3
 	accurate_range = 5
-	damage = 25
+	damage = 40
 	penetration = 100
 	sundering = 7
 	max_range = 30
 
 /datum/ammo/bullet/dual_cannon/on_hit_turf(turf/T, obj/projectile/P)
-	P.proj_max_range -= 20
+	P.proj_max_range -= 5
```

Comment:
this meants you can pierce like 6-7 turfs without issue, which is probably not good.